xml/deployment_sysreqs.xml

```diff
@@ -744,4 +748,4 @@
    </itemizedlist>
   </sect2>
  </sect1>
- </chapter>
+</chapter>
```

Comment:
Allrightythen, I probably missed a few things, but hopefully this helps. Nice job keeping up with a moving target.

Reply:
I did what i could as well, adde a few hints. Overall, good job Liam!